assets/src/editor/events/tickets/editor-ticket/list-view/editor-ticket-entity-list-item.js

```diff
@@ -86,18 +86,22 @@ class EditorTicketEntityListItem extends Component {
 					</div>
 					<div className="ee-ticket-list-item">
 						<span className="ee-ticket-list-item-label">
-							{ __( 'Start Date:', 'event_espresso' ) }
+							{ __( 'Price:', 'event_espresso' ) }
 						</span>
 						<span className="ee-ticket-list-item-value">
-							{ ticketEntity.startDate.toFormat( 'ddd MMM YY h:mm a' ) }
+							{
+								ticketEntity.price.formatter.formatNumber(
+									ticketEntity.price.amount.toNumber()
+								)
```

Comment:
You should be able to just use `ticketEntity.price.toString()` to output the formatted value?

Reply:
does that apply all of the currency symbol formatting, etc? 
if so, then ya, I'll do that, thnx

Reply:
It should:
```js
toString() {
		return this.formatter.format(
			this.amount.toNumber(),
			this.formatter.settings
		);
	}
```

Also worth noting that this is a "magic" method.  If you concatenate the object with a string (or cast it to a string) it will call `toString()` and output the formatted money value.